cli/app/app.go

```diff
@@ -854,8 +854,25 @@ func loadComposeFile(file string, opt convertOptions) KomposeObject {
 
 	// transform composeObject into komposeObject
 	composeServiceNames := composeObject.ServiceConfigs.Keys()
+
+	// volume config and network config has not supported
+	if len(composeObject.NetworkConfigs) > 0 {
+		logrus.Warningf("Unsupported network configuration of compose v2 - ignoring")
+	}
+	if len(composeObject.VolumeConfigs) > 0 {
+		logrus.Warningf("Unsupported volume configuration of compose v2 - ignoring")
+	}
+
+	count := 0
 	for _, name := range composeServiceNames {
 		if composeServiceConfig, ok := composeObject.ServiceConfigs.Get(name); ok {
+			//FIXME: networks always contains one default element, even it isn't declared in compose v2.
+			if len(composeServiceConfig.Networks.Networks) > 0 &&
+				strings.Compare(composeServiceConfig.Networks.Networks[0].Name, "default") != 0 &&
```

Comment:
But users can configure their default network https://docs.docker.com/compose/networking/#/configuring-the-default-network and we should give them warnings for that too